orchagent/aclorch.h

```diff
@@ -514,7 +520,8 @@ class AclOrch : public Orch, public Observer
     bool m_isCombinedMirrorV6Table = true;
     map<string, bool> m_mirrorTableCapabilities;
     map<acl_stage_type_t, bool> m_L3V4V6Capability;
-
+    map<string, string> m_switchMetaDataCapabilities;
```

Comment:
Why not use `map<string, bool> m_switchMetaDataCapabilities`?

Reply:
because i am storing the range values as well